main/main.go

```diff
@@ -162,11 +164,23 @@ func reportExecutionStatus() {
 
 func install() {
 	operation := "install"
+	path, _ := os.Getwd()
+	_, err := exec.Command("/bin/sh", filepath.Join(path, "scripts/service_install.sh")).Output()
+	if err != nil {
+		errorMessage := fmt.Sprintf("Error installing service: %+v", err)
+		errorLogger.Println(errorMessage)
+	}
 	testCommand(operation)
 }
 
 func enable() {
 	operation := "enable"
+	path, _ := os.Getwd()
+	_, err1 := exec.Command("/bin/sh", filepath.Join(path, "scripts/service_enable.sh")).Output()
```

Comment:
NIT: If you make the file executable (check in the file with the +x bit set), you wouldn't need to specify `/bin/sh` here.